webpack/dev.js

```diff
@@ -16,7 +16,7 @@ const dev = webpackMerge(base, {
   entry: './src/index.jsx',
 
   // Enables source maps that can be accessed in browser dev tools
-  devtool: 'source-map',
+  devtool: 'cheap-module-eval-source-map',
```

Comment:
Source maps are used to translate the lines of the bundle created by Webpack to lines in the individual files that comprise it. This change reduces the cost of creating the source maps.